### PR TITLE
screen-dimming never turns off

### PR DESCRIPTION
**Pull request by meets2tarun**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
#88 
This PR will take care to never turns the screen off.
### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by xeon-zolt** [comment]
@meets2tarun please instead of making pull request based on meilix-default settings package, we should complete the script for building the package in CI first

**Comment by tabesin** [comment]
Please also consider to update the ~/.xsession file or whatever /etc/X11/Xsession defines as USERXSESSION.

File: build.sh [hunk 1]
```diff
@@ -137,6 +137,9 @@ apt-get -q -y --purge install --no-install-recommends libreoffice-gtk libreoffic
 dpkg -i meilix-imclient_*_all.deb
 apt-get install -f
 
+#screen-dimming turns off always
+echo -ne "\033[9;0]" >> /etc/issue
```
Comment (abishekvashok) [inline build.sh]: Consider `setterm -blank 0 >> /etc/issue` as well.

**Comment by meets2tarun** [comment]
> Please also consider to update the ~/.xsession file or whatever /etc/X11/Xsession defines as USERXSESSION.

@tabesin 
Which file in `Xsession.d` ?

**Pull request merged**